server/vulnerabilities/cpe.go

```diff
@@ -254,7 +254,7 @@ func TranslateSoftwareToCPE(
 			continue
 		}
 		if cpe == "" {
-			continue
+			cpe = fmt.Sprintf("none:%d", software.ID)
```

Comment:
Also, let's add a comment on why we are doing this, as it will be confusing for future readers.